CMakeLists.txt

```diff
@@ -28,6 +28,16 @@ elseif ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
     if (CMAKE_CXX_COMPILER_VERSION VERSION_LESS 3.3)
         message(FATAL_ERROR "Clang version must be at least 3.3!")
     endif()
+elseif ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC")
+    # allow MSVC VS2015.1 and VS2013 toolchain ( VS2013 is not tested )
```

Comment:
ISTR this PR predates our switch to C++-14. I suppose VS2013 does not support C++-14, so perhaps this check can be simplified?